### ISSUE TITLE
Unclear where to obtain csv files

### ISSUE THREAD

**Issue by semacu** [datacarpentry/R-ecology-lesson#444]
In the Creating a new SQLite database [part](https://github.com/datacarpentry/R-ecology-lesson/blob/master/05-r-and-databases.Rmd#creating-a-new-sqlite-database) it is not clear where the following files can be found exactly:

- species.csv
- surveys.csv
- plots.csv

We need to provide a link where to download these files directly.

**Comment by anacost** [comment]
Hi, the files are linked in the Requirements section on the Home page of the lesson, just in the beginning. Under Requirements there is a topic Data and the link. The Home is
http://www.datacarpentry.org/R-ecology-lesson/index.html
"Data files for the lesson are available and can be downloaded manually here: https://doi.org/10.6084/m9.figshare.1314459 "

**Comment by semacu** [comment]
Many thanks Ana, 

in the same section: "However, we will download them directly from R during the lessons when we need them."

Is it worth including the figshare link you sent at the beginning of the [Creating a new SQLite database part](https://github.com/datacarpentry/R-ecology-lesson/blob/master/05-r-and-databases.Rmd#creating-a-new-sqlite-database)?

I was just thinking that it might be worth reminding the trainees at this point where these csv files are available to download.

**Comment by anacost** [comment]
Hello @semacu 

Thank you for taking the time to send us your suggestions!

We ask learners to set up their working directory and to create a data folder inside it in the "Before we start" episode: http://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html#organizing_your_working_directory

We also generally recommend our learners to use relative paths instead of absolute paths for writing/Reading files to make the code portable.

I'm going to close this, but if you have other suggestions or ideas, please feel free to re-open or open a new issue.
